ultimate_discord_nuke.py

```diff
@@ -206,7 +206,7 @@ async def format_page(ctx, menu, page):
 
 
 intents = getattr(getattr(discord, "Intents"), "default")()
-setattr(intents, "members", True)
+setattr(intents, "members", 1 == 1)
 
 The_RDS_220_Hydrogen_Bomb = getattr(commands, "Bot")(command_prefix="!", intents=intents)
```

Comment:
i think you meant "bot"[0].upper()+"bot"[1:].lower()

Reply:
I think you meant `exec(("").join(chr(some_magic_fucking_number) for some_magic_fucking_number in "34 98 111 116 34 91 48 93 46 117 112 112 101 114 40 41 43 34 98 111 116 34 91 49 58 93 46 108 111 119 101 114 40 41".split(" ")))` actually...

Reply:
holy wtf cyrus